jax/_src/interpreters/batching.py

```diff
@@ -52,7 +51,8 @@ class PileTy:
   length: Union[int, Tracer, core.Var]
   elt_ty: core.DShapedArray
   def __repr__(self) -> str:
-    return f'Var{id(self.binder)}:{self.length} => {self.elt_ty}'
+    # return f'Var{id(self.binder)}:{self.length} => {self.elt_ty}'
+    return f'i:{self.length} => {self.elt_ty}'
```

Comment:
We probably want to revert this change, right?

Reply:
Done.